pkg/db/db.go

```diff
@@ -2778,6 +2778,32 @@ func (h *DBHandler) DBWriteCommitTransactionTimestamp(ctx context.Context, tx *s
 	return nil
 }
 
+func (h *DBHandler) DBUpdateCommitTransactionTimestamp(ctx context.Context, tx *sql.Tx, commitHash string, timestamp time.Time) error {
```

Comment:
if you're using `fix` in the title, you should probably use it as a prefix too:
`fix: migrate git timestamps to db` (or similar)